speechbrain/pretrained/interfaces.py

```diff
@@ -592,12 +676,133 @@ def transcribe_batch(self, wavs, wav_lens):
         with torch.no_grad():
             wav_lens = wav_lens.to(self.device)
             encoder_out = self.encode_batch(wavs, wav_lens)
-            predicted_tokens, scores = self.mods.decoder(encoder_out, wav_lens)
-            predicted_words = [
-                self.tokenizer.decode_ids(token_seq)
-                for token_seq in predicted_tokens
-            ]
-        return predicted_words, predicted_tokens
+            if detail_level == InferenceDetailLevel.TOP1_HYP:
```

Comment:
I like the idea, but this would be something to apply to all interfaces :/ And maybe we should discuss the format of it as well, because it's making the code of the interface MUCH longer.

Reply:
yeah, well, testing here ^^"
interface needs refactoring...

Reply:
put it in an extra file - available as Easter Egg - if one wants & desires to extend (it's in another "domain")